sap/5.1/modules/ROOT/pages/sap-connector-studio.adoc

```diff
@@ -123,6 +123,19 @@ image::sap-connector-configuration-03-finish.png[]
 image::sap-connector-configuration-02.png[]
 +
 . In the *Install* menu, click *Browse* to locate the library file.
++
+[NOTE]
+====
+On MacOS, starting with JCO version 3.1.2, the native library extension was changed from `jnilib` to `dylib`. Anypoint Studio versions 7.0 through 7.6 have a limitation on selecting the `dylib` file type. After installing the library from the local machine:
+
```

Comment:
Looking at this locally, I'm not sure the formatting is correct.

![image](https://user-images.githubusercontent.com/18625575/92670055-046f3780-f2c8-11ea-9c60-0b7cf4630250.png)

Reply:
Are these supposed to be sublists under `3. In the *Install* menu, click *Browse* to locate the library file.`

Reply:
If you want them as sub-bullets and you want the Note at the top, there is a way to do that, but I'd have to play with it. I think you can just indent them, but I'd have to verify.

Reply:
Other than the formatting issue, the content looks good.